projects/telemetry/src/main.c

```diff
@@ -0,0 +1,53 @@
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+
+#include "gpio.h"        // General Purpose I/O control.
+#include "soft_timer.h"  // Software timers for scheduling future events.
+#include "interrupt.h"   // For enabling interrupts.
+#include "gps.h"
+
+static const GPIOAddress pins[] = {
+  {.port = GPIO_PORT_A, .pin = 2 },   //
+  {.port = GPIO_PORT_A, .pin = 3 },   //
+};
+
+static UARTSettings s_settings = {
```

Comment:
Should be const unless I'm missing something.

Reply:
Done